src/frontend/src/app/core/ui/form-field/form-field.component.ts

```diff
@@ -98,10 +99,16 @@ export class FormFieldComponent implements AfterContentInit {
       .subscribe(() => {
         this.changeDetectorRef.markForCheck();
       });
+
+    this.inputDirective.touchChange
```

Comment:
Since form errors are only shown when the input is blurred or the value changes, we need to run change detection whenever the input is blurred